go/apps/bulk_message/vumi_app.py

```diff
@@ -55,6 +57,18 @@ def on_window_cleanup(self, window_id):
     def get_window_id(self, conversation_key, batch_id):
         return ':'.join([conversation_key, batch_id])
 
+    def _send_progress_key(self, conv):
+        return ':'.join([self.worker_name, conv.key, conv.batch.key])
```

Comment:
What happens if someone initiates two bulk sends from the same conversation in a row with different text?